cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -57,28 +48,56 @@ func K8sCodegen() error {
 
 	log.Infof("Running deepcopy code-generation for Custom Resource group versions: [%v]\n", gvb.String())
 
-	fdc := func(a string) error { return deepcopyGen(binDir, repoPkg, a, gvMap) }
-	if err = withHeaderFile(fdc); err != nil {
+	apisPkg := filepath.Join(repoPkg, scaffold.ApisDir)
+	fqApis := createFQAPIs(apisPkg, gvMap)
+	f := func(a string) error { return deepcopyGen(a, fqApis) }
+	if err = withHeaderFile(f); err != nil {
```

Comment:
While we're here, can we update the name for [`withHeaderFile()`](https://github.com/operator-framework/operator-sdk/blob/c447c45f49f7075fe041ba910b632c10cc00bc8f/cmd/operator-sdk/internal/genutil/genutil.go#L113)?
`withHeaderFile` isn't very descriptive on first read so maybe something like:
```Go
func generateWithHeaderFile(generatorFunc func(string) error) (err error) {
```
And probably some Godocs for the arguments.

Or you can put a TODO if you want to keep that refactoring separate from this PR.